doc/cdi-operator-olm.md

```diff
@@ -52,7 +63,41 @@ cdi-7c7fc4f774-bdbsh                   1/1     Running   0          37s
 marketplace-operator-d8cc985d4-mv7xp   1/1     Running   0          2m40s
 
 ```
-### CDI installation by means of OLM and marketplace operators
+### CDI installation via OLM and marketplace operators 
+#### OKD4.x cluster
+- Install CDI operatorsource manifest that specifies the location of CDI OLM bundle in quay
+```bash
+kubectl apply -f _out/manifests/release/olm/cdi-operatorsource.yaml
+```
+- Create CDI namespace
+```bash
+kubectl create ns cdi 
+```
+- Configure namespace to be allowed to create operators there
+```bash
+kubectl apply -f _out/manifests/release/olm/operatorgroup.yaml
+```
+- Install subscription that will point from which channel the app is downloaded
+```bash
+kubectl apply -f  _out/manifests/release/olm/cdi-subscription.yaml
+```
+- Verify CDI installation plan was created
+```bash
+kubectl get operatorsource,catalogsourceconfig,catalogsource,subscription,installplan -n cdi
+NAME                                    PACKAGE   SOURCE   CHANNEL
+subscription.operators.coreos.com/cdi   cdi       cdi      beta
+
+NAME                                             CSV                 SOURCE   APPROVAL    APPROVED
+installplan.operators.coreos.com/install-995l9   cdioperator.0.0.0            Automatic   true
+
+```
+- Now cdi-operator starts running but in order for it to succeed we need to deploy cdi cr
```

Comment:
"but in order for it to succeed" -> "but in order to install CDI"
the operator itself is fine without the CR

Reply:
Done. thanks